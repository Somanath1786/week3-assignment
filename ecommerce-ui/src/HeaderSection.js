// Define your imports here:
import React from 'react';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';
import { withStyles } from '@material-ui/core/styles';
import { lightBlue } from '@material-ui/core/colors';
import { Card } from '@material-ui/core';

const styles = {
    root: {
        flexGrow : 1,
    },

    appbar : {
        alignItems : 'center',
        background: lightBlue[200]      
    },
};

class HeaderSection extends React.Component {
    render() {
        const {classes} = this.props;
        return (
            <div className={this.props.classes.root}> 
            <Card> 
                <AppBar 
                    className={this.props.classes.appbar}
                    position="static" color="default">
                    <Toolbar>
                        <Typography                             
                            variant="h4" color="inherit ">
                            Soma's Vacation Rentals
                        </Typography>
                    </Toolbar>                    
                </AppBar>
                </Card>               
            </div>
        );
    };
}

export default withStyles(styles)(HeaderSection);