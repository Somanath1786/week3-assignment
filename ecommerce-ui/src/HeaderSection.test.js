import React from 'react';
import ReactDOM from 'react-dom';
import {shallow, render, mount} from 'enzyme';
import HeaderSection from './HeaderSection';

it('shallow renders', () => {
    const wrapper = shallow(<HeaderSection />);
    expect(wrapper).toMatchSnapshot();
  
    expect(wrapper.props('classes')).toBeDefined();
    
  });