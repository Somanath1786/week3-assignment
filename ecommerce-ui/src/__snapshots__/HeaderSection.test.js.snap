// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow renders 1`] = `
<HeaderSection
  classes={
    Object {
      "appbar": "HeaderSection-appbar-2",
      "root": "HeaderSection-root-1",
    }
  }
/>
`;
